### DISCUSSION TITLE
Is it possible to open multiple instance from different app ?

### DISCUSSION THREAD

**Discussion by ziaratban** [isar/isar#104] [Help]
Hi
I have several apps and I want to store all the data of these apps in one database.(eg one database and multiple instance).

Is it possible to open multiple instance in different app ?
Does isar support?

**Comment by anandsubbu007** [comment]
Any update around this question?

If yes, 
what happens when we store data at the same time?
How read & write transaction works?
How to manage it?
How id will be generated?

**Reply by simc** [reply]
It is not possible, sorry.
